.evergreen/config.yml

```
@@ -1071,6 +1071,25 @@ functions:
               --ca_file "${PROJECT_DIRECTORY}/testdata/kmip-certs/ca-ec.pem" \
               --cert_file "${PROJECT_DIRECTORY}/testdata/kmip-certs/server-ec.pem"
           fi
+    # Wait up to 10 seconds for the KMIP server to start.
+    - command: shell.exec
+      params:
+        script: |
```

Optional: Kevin recently added a change to explicitly specify the shell in all commands. I'm not completely sure the intended effect, but it seem to make sense to maintain that pattern.

```suggestion
        shell: "bash"
        script: |
```